docs.md

```diff
@@ -0,0 +1,1366 @@
+- We can also have interactive coding cells in modules using AlaSQL, which is our approach for SQL modules, for example: [SQL Basics](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/sql_basics/sql_basics.md)
+
+We don't have a way to include interactive code for git or bash. For those modules, we just instruct learners to work on their own machines.
+
+### Interactive R
+
+If your lesson includes R code examples hosted in binder from [our R notebook repository](https://github.com/arcus/education_r_environment), then you'll need to load the R macros by adding the following `import` statement to the front matter of your module:
+
+```
+import: https://raw.githubusercontent.com/arcus/education_modules/templates_update/_module_templates/macros_r.md
+```
+
+You can then insert the `@lesson_prep_r` macro, which includes instructions for opening up the relevant rmd document, in your Lesson Preparation section:
+
+```
+## Lesson Preparation: R Studio
+
+@lesson_prep_r
+```
+
+Note that you must have `r_file` filled out in the front matter. 
+
+### Interactive Python
+
+If your lesson includes interactive python code in sagemath cells, then you'll need to load the Python macros by adding the following `import` statement to the front matter of your module: 
+
+```
+import: https://raw.githubusercontent.com/arcus/education_modules/templates_update/_module_templates/macros_python.md
+```
+
+You can then insert the `@lesson_prep_python` macro, which includes instructions and an example interactive code block, in your Lesson Preparation section:
+
+```
+## Lesson Preparation: Interactive Python
+
+@lesson_prep_python
+```
+
+### Interactive SQL
+
+If your lesson includes interactive SQL code, you'll need to load the SQL macros. Each table is generated, row by row, in the macros for that lesson, so to avoid loading large macros unnecessarily, there are several different SQL macro modules available:
+
+ - `macros_sql.md` is required for any module with interactive SQL
+ - `macros_sql_table_allergies.md` loads the allergies table
+ - `macros_sql_table_patients.md` loads the patients table
+ - `macros_sql_table_observations.md` loads the observations table
+
+Note that very small tables (just a few rows) can be constructed right in the front matter for the module that uses them. Any larger tables, or tables that need to be re-used across different modules, should be created in separate table macro modules, though.
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+To learn about creating a new macro, such as a new SQL table, see our [instructions for DART macros](https://github.com/arcus/education_modules/blob/main/macros_instructions.md).
```

Comment:
This link doesn't yet exist (I think).

Reply:
Indeed it doesn't --- I'll change it to the temporary link on the template_updates branch for clarity